src/pages/Content/index.js

```diff
@@ -31,7 +31,22 @@ function click(selector) {
 }
 // The function plays a recording when needed
 function playRecording(recordedEvents) {
-  recordedEvents.forEach(element => {
-    click(element);
-  });
+  let intervalId = setInterval(() => {
```

Comment:
This should be a separate function `tryClickUntilExists(selector, interval, maxRetries)`

Reply:
Created a new function as requested.